examples/panorama/panorama.py

```diff
@@ -0,0 +1,16 @@
+#Simply import of "panorama-module.py" and you can use each function by calling it with name of the drone inside arguments.
+from djitellopy import Tello
+import cv2
+import time
+import panoramaModuled
```

Comment:
`ModuleNotFoundError: No module named 'panoramaModuled'`

Reply:
Yes, sorry I wanted to match name convention and forgot to change it. Thank you!

Reply:
Now it should be OK.